client/.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
add more things to .gitignore use this https://www.toptal.com/developers/gitignore